packages/fxa-auth-server/lib/payments/utils.ts

```diff
@@ -8,3 +8,7 @@ export function generateIdempotencyKey(params: string[]) {
   sha.update(params.join(''));
   return sha.digest('base64url');
 }
+
+export function addTimeToIdempotencyKey() {
```

Comment:
Could we improve this function a little bit? At the moment, if the first request came in at 40min and 59seconds, and the second request came in 1 second later at 41min and 0 seconds, this function would return different values. (Additionally every hour, this function would return the same value. 8h40min and 9h40min both return 40)

Preferably this function would return the same value for a 10s to 1min window.